metadata_tools/columns.py

```
@@ -26,4 +26,4 @@ def get_bodies(body_names):
 #############################################
 column_files = list(defs.COLUMN_DIR.glob('COLUMNS_*.py'))
 for file in column_files:
-    exec(open(file).read())
+    exec(open(file.as_posix()).read())
```

Ruff (0.14.10)

Use of `exec` detected

(S102)